lib/ibm_watson/visual_recognition_v3.rb

```diff
@@ -195,10 +190,12 @@ def classify(images_file: nil, accept_language: nil, url: nil, threshold: nil, o
     #
     #   You can also include images with the **images_file** parameter.
     # @param images_file_content_type [String] The content type of images_file.
+    # @param accept_language [String] The desired language of parts of the response. See the response for details.
     # @param images_filename [String] The filename for images_file.
     # @return [DetailedResponse] A `DetailedResponse` object representing the response.
-    def detect_faces(images_file: nil, url: nil, images_file_content_type: nil, images_filename: nil)
+    def detect_faces(images_file: nil, url: nil, images_file_content_type: nil, accept_language: nil, images_filename: nil)
```

Comment:
Is this a breaking change? Should `accept_language` be moved after `images_file`?